include/material/norsand.h

```diff
@@ -0,0 +1,165 @@
+#ifndef MPM_MATERIAL_NOR_SAND_H_
+#define MPM_MATERIAL_NOR_SAND_H_
+
+#include <iostream>
+#include <limits>
+
+#include <cmath>
```

Comment:
This should go above c++ includes

Reply:
Clang does not do this? I will make them on top then.